src/scripts/runtests.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+import sys
+import os
+import argparse
+import subprocess
+import re
+import settings
+from colors import bcolors
+
+def convert_if_int(text):
+    return int(text) if text.isdigit() else text
+
+def natural_keys(text):
+    return [ convert_if_int(c) for c in re.split('(\d+)', text) ]
+
+def get_arguments():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-q", help = "only shows the final stats", action='store_true')
+    parser.add_argument("--test-folder", help = "the folder of the tests to run", default = settings.NUMBER_DIR)
+    parser.add_argument("--engine-location", help = "the location of javascript engine", required = True, default = "")
+
+    if len(sys.argv) <= 1:
+        parser.print_help()
+        sys.exit(1)
+
+    script_args = parser.parse_args()
+
+    return script_args
+
+def run_tests(options):
+    COUNTER = 0  # The count of tests run
+    FAIL = 0 # The count of failed tests
+    PASS = 0 # The count of passed tests
+    TEST_FILES = os.listdir(options.test_folder) #Files to test
+
+    TEST_FILES.sort(key = natural_keys)
+
+    if (not options.q):
+        print("%s\nTest\t\t\tExit code\n%s" % (bcolors.OKBLUE, bcolors.ENDC))
+
+    for filename in TEST_FILES:
+        if filename.endswith(".js"):
+            test_file = os.path.join(options.test_folder, filename)
+            p = subprocess.Popen([options.engine_location, test_file], stderr = open(os.devnull, 'w'))
+            p.communicate()
+            COUNTER += 1
+            if (not p.returncode):
+                color = bcolors.OKGREEN
+                PASS += 1
+            else:
+                color = bcolors.FAIL
+                FAIL += 1
+            if (not options.q):
+                print("%s%s   \t%d%s" % (color , filename, p.returncode, bcolors.ENDC))
+
+    if (not COUNTER):
+        print("%sThere are no .js files in the given test folder.%s" % (bcolors.OKBLUE, bcolors.ENDC))
+        return
+
+    pass_percentage = (PASS * 100.0) / COUNTER
+    fail_percentage = (FAIL * 100.0) / COUNTER
+
+    print("\n%sPASS:\t%d\t%f %%%s" % (bcolors.OKGREEN, PASS, pass_percentage, bcolors.ENDC))
+    print("%sFAIL:\t%d\t%f %%%s" % (bcolors.FAIL, FAIL, fail_percentage, bcolors.ENDC))
+
+def main(options): # Main function
```

Comment:
Will this `main` function be exported to somewhere else? If no, please avoid creating this kind of one-call functions. Remove it, please!